### DISCUSSION TITLE
Sujet / Thema 3 –– Datenbestimmungen / Conditions d'utilisation

### DISCUSSION THREAD

**Discussion by Fijord** [GeoUnconference/discussions#16] [Ideen / Idées]
Es wäre einen grossen Vorteil, wenn OSM ohne grosse Mühe, die Geodaten von swisstopo, Kantonen, Städte gebrauchen könnte.

**Comment by hinjud** [comment]
Was genau ist mit "ohne grosse Mühe" gemeint?

**Reply by rastrau** [reply]
Ich meine, ein häufiges Problem ist, dass Nutzungsbedingungen erfordern, dass die Datenquelle *im Kontext der Nutzung* genannt werden muss. Das ist für OSM nicht machbar, weshalb eine summarische Nennung von Datenquellen erfolgt.

**Reply by rastrau** [reply]
Eine Übersicht kompatibler und nicht-kompatibler Schweizer Datenquellen findet man hier: https://wiki.openstreetmap.org/wiki/Switzerland/Datasources. 

Bei den [«Not-compatible data sources»](https://wiki.openstreetmap.org/wiki/Switzerland/Datasources#Not-compatible_data_sources) sind einige **Hinderungsgründe** aufgeführt. Bezüglich swisstopo ist dort vermerkt, was ich oben wiederzugeben versucht habe.

**Comment by simonpoole** [comment]
Siehe https://sosm.ch/use-of-swisstopo-data-and-products-with-and-in-openstreetmap/

Das hat natürlich nur einen Bezug auf die grundsätzliche Nutzbarkeit von Daten, nicht ob die Nutzung überhaupt sinnvoll oder gewünscht ist.

**Comment by Fijord** [comment]
Es gibt immer wieder Diskussionen, ob es erlaubt ist, diese Geodaten für OSM zu gebrauchen. Wie zum Beispiel die AV der Zentralschweiz und der West- und Ostschweiz, siehe [https://s.geo.admin.ch/91edf397a4](url) oder Höhenlinie.

**Comment by rastrau** [comment]
Für Daten von swisstopo gab es am 14.09.2021 erfreuliche Neuigkeiten, cf:
- [Tweet der Swiss OpenStreetMap Association](https://twitter.com/SwissOSM/status/1437782493084717056)
- [Eintrag in der `talk-ch` Mailing-Liste](http://lists.openstreetmap.ch/pipermail/talk-ch/2021-September/011262.html)
- [Website von swisstopo zum Thema](https://www.swisstopo.admin.ch/de/home/meta/konditionen/geodaten/source.html#ui-collapse-46) (spezifisch: Punkt 6; Hervorhebungen durch mich):


> Wenn swisstopo-Geodaten oder auf swisstopo-Geodaten basierende Daten in einer Datenbank mit vielen verschiedenen Quellen enthalten sind, **genügt es, swisstopo in der Liste der Quellen dieser Datenbank zu erwähnen**. Die **explizite Erwähnung von swisstopo bei jeder Nutzung der Daten ist nicht mehr erforderlich**, auch nicht durch den Lizenznehmer der betreffenden Datenbank. In diesem Fall genügt ein indirekter Hinweis auf den Herausgeber der Datenbank, der selbst swisstopo erwähnt.

**Comment by davidoesch** [comment]
swisstopo hat nun die Anforderungen bezüglich der Quellenangabe geklärt und dies auf einer FAQ-Webseite dokumentiert. 

https://www.swisstopo.admin.ch/de/home/meta/konditionen/geodaten/source.html 

Für den Punkt 6 hat sich swisstopo an der Lösung von opentransportdata.swiss orientiert. (Ausnahmeregelung für Datenbankwerke). 

**-> Damit sollte die Nutzung von swisstopo-Geodaten durch die OSM-Community weitgehend möglich sein.**

**Reply by RafBov** [reply]
For question about the FAQ : geodata@swisstopo.ch

**Comment by sfkeller** [comment]
@RafBov wrote:

> For question about the FAQ : geodata@swisstopo.ch

Is this e-mail also a sort of "official" point of contact at swisstopo about questions related to OpenStreetMap? 

I'm referring to a recommendation from the study from Hitz & Stürmer (to be published 2021) which proposes that data owners should "designate a central point of contact for OpenStreetMap issues both externally and internally".

**Reply by RafBov** [reply]
It's a point of contact for question regarding the use of "Geobasisdaten des Bundesrechts in der Verantwortung von swisstopo" .  It can also be questions about the content of the data itself. geodata@swisstopo.ch will answer itself or dispatch the question to the right person.  It also should meet the needs you talk about "a central point of contact for OpenStreetMap issues both externally and internally". What do you understand with  both externally and internally ? Could you explain ?

**Reply by sfkeller** [reply]
> What do you understand with both externally and internally ?

That's IMHO simply meaning that this mailadress (and person(s) behind) this point of contact serve for questions from outside an organization and from inside as well.

**Reply by rastrau** [reply]
Interestingly, this relates to parts of the discussion in https://github.com/GeoUnconference/discussions/discussions/5.

**Comment by rastrau** [comment]
This topic was touched upon in GeoUnconference 3 while a group discussed topic 3: https://github.com/GeoUnconference/discussions/discussions/18#discussioncomment-2834135

**Reply by sfkeller** [reply]
Thx! FYI: The NGDI POP study has just been published: See the link and the summary here https://www.geowebforum.ch/thread.php?postingID=4805. 

The study concludes with the following five recommendations to government agencies:

1. government agencies should use OpenStreetMap data where it makes sense. This is allowed without having to change your license if you know the options.
2. authorities should contact the SOSM association if they have questions about the use of OpenStreetMap data.
3. authorities should clarify whether engagement with OpenStreetMap is beneficial to them, such as membership in SOSM.
4. build internal capacity for OpenStreetMap (e.g., co-organize mapathons or participate in mapping parties).
5. the authorities should designate a central point of contact for questions about OpenStreetMap, both for external and internal inquiries (analogous to the "Leitfaden" /guide of the University of Bern).
